client/src/app/shared/validator/validator.module.ts

```diff
@@ -7,7 +7,6 @@ import { ValidatorService } from './validator.service';
 @NgModule({
   imports: [CommonModule],
   exports: [ValidatorComponent],
-  declarations: [ValidatorComponent],
-  providers: [ValidatorService],
+  declarations: [ValidatorComponent]
```

Comment:
removed the provided Validator service as it just has static methods and is not injectable